Classification/DecisionTree/src/main/java/org/tribuo/classification/dtree/CARTClassificationOptions.java

```diff
@@ -69,10 +73,12 @@ public CARTClassificationTrainer getTrainer() {
         CARTClassificationTrainer trainer;
         switch (cartTreeAlgorithm) {
             case CART:
-                if (cartSplitFraction <= 0) {
-                    trainer = new CARTClassificationTrainer(cartMaxDepth, cartMinChildWeight, 1, impurity, cartSeed);
+                if (cartSplitFraction == 0) {

```

Comment:
Probably best to set the default value of cartSplitFraction to 1.0 and then remove this if statement entirely.

Reply:
done